results_dashboard.py

```diff
@@ -1,24 +1,31 @@
-from app import create_app, db
-from werkzeug.middleware.profiler import ProfilerMiddleware
-# from flask_migrate import Migrate, upgrade
+from app import create_app
+from werkzeug.contrib.profiler import ProfilerMiddleware
 import os
-from dotenv import load_dotenv
+import glob
 
+from app.models.sqlalchemy import DataStore
 
-dotenv_path = os.path.join(os.path.dirname(__file__), '.env')
-if os.path.exists(dotenv_path):
-    load_dotenv(dotenv_path)
 
 # Determine config settings from environment variables.
 config_name = os.getenv('FLASK_CONFIG') or 'development'
 
 app = create_app(config_name)
 
-# if app.config['TESTING']:
-#     app.wsgi_app = ProfilerMiddleware(app.wsgi_app, restrictions=[30])
+def create_datastore(location):
 
-# migrate = Migrate(app, db)
+    location = os.path.abspath(location)
+    datastore_location = os.path.join(location, 'molssi_jobstore.db')
+    db = DataStore(datastore_location)
 
+    for job in os.listdir(location):
+        job_path = os.path.abspath(os.path.join(location, job))
+        
+        if os.path.isdir(job_path):
+            db.add_job(job_path)
+    
+    db.Session().close()
```

Comment:
To be consistent, you could move this to the  same file `app/__init__py` where it has the `create_app` factory method too.